projects/plugins/jetpack/tools/webpack.config.extensions.js

```diff
@@ -173,6 +168,9 @@ module.exports = [
 			'editor-experimental': editorExperimentalScript,
 			'editor-beta': editorBetaScript,
 			'editor-no-post-editor': editorNoPostEditorScript,
+			// this entry is added to generate components.css which is enqueued at
```

Comment:
I assume we'll need to copy the contents of `projects/plugins/jetpack/extensions/shared/components/style.scss` in there as well so they can be part of the build?

Reply:
But those class names are not used anywhere directly in the Jetpack Plugin. Some JS files in the jetpack_vendor folder are using them though. Should I move ahead and add them? I am not sure about the right way to confirm if they are required anymore or not @jeherve

Reply:
I don't believe we use the classes ourselves directly, but we use the core `Warning` component, which when displayed in the editor context includes CSS (that we try here to copy so it can be enqueued on the frontend as well):
https://github.com/WordPress/gutenberg/blob/2e4e0796ccab8455eab1467c9cffcd3ff69e6c40/packages/block-editor/src/components/warning/style.scss

Reply:
Copied over the contents of `projects/plugins/jetpack/extensions/shared/components/style.scss` to `projects/plugins/jetpack/extensions/shared/components/upgrade-nudge/style.scss `so that the relevant css can be generated in the` components.css` file 
@jeherve